internal/pkg/deploy/cloudformation/stack/testdata/workloads/rdws-manifest.yml

```diff
@@ -52,6 +52,8 @@ environments:
     command: crazy diamond
     variables:
       LOG_LEVEL: debug        # Log level for the "test" environment.
+      DB_NAME:
```

Comment:
This change and `svc-manifest`'s `nginx` sidecar manifest change are not reflected in   `rdws-test.stack.yml` or `svc-prod.stack.yml` respectively. They are kept out by the feature flag. This is to show that they are indeed kept out.